darwin/future/tests/meta/objects/test_itemmeta.py

```diff
@@ -44,3 +44,26 @@ def test_delete(item: Item) -> None:
             json={},
         )
         item.delete()
+
+
+def test_restore(item: Item) -> None:
+    with responses.RequestsMock() as rsps:
+        team_slug = item.meta_params["team_slug"]
+        dataset_id = item.meta_params["dataset_id"]
+        rsps.add(
+            rsps.POST,
+            item.client.config.api_endpoint + f"v2/teams/{team_slug}/items/restore",
+            status=200,
+            match=[
+                json_params_matcher(
+                    {
+                        "filters": {
+                            "item_ids": [str(item.id)],
+                            "dataset_ids": [dataset_id],
+                        }
+                    }
+                )
+            ],
+            json={},
+        )
+        item.restore()
```

Comment:
As with the other one, I would sad path these, but in these, they are small enough, that it isn't a massive deal.  Red/Green/Refactor is a good approach.

Reply:
Quick Q on this - Since neither `item.restore()` or `item_query.restore() `take any parameters nor do they return anything, what could we consider the sad path?

Reply:
@owencjones I discussed these tests with Nathan - Since they don't return anything or take any arguments, there should only be one path these functions can actually take

Error responses from the API are covered in the underlying Core method tests. Given this, I'll proceed to merge but let me know if you disagree or have any other thoughts